src/main/java/com/datastax/oss/protocol/internal/response/error/CASWriteUnknown.java

```diff
@@ -0,0 +1,78 @@
+/*
+ * Copyright DataStax, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.datastax.oss.protocol.internal.response.error;
+
+import com.datastax.oss.protocol.internal.Message;
+import com.datastax.oss.protocol.internal.PrimitiveCodec;
+import com.datastax.oss.protocol.internal.PrimitiveSizes;
+import com.datastax.oss.protocol.internal.ProtocolConstants;
+import com.datastax.oss.protocol.internal.response.Error;
+
+public class CASWriteUnknown extends Error {
+
+  /** The consistency level of the query that triggered the exception. */
+  public final int consistencyLevel;
+  /** The number of nodes having answered the request. */
+  public final int received;
+  /**
+   * The number of replicas whose response is required to achieve {@code consistencyLevel}.
+   *
+   * <p>It is possible to have {@code received >= blockFor} if {@code data_present} is false. Also
```

Comment:
There is no `data_present` field in this class. This comment might not apply here.

Reply:
Bah, good catch; copy/paste error.  Will fix.

Reply:
Fixed